unturned/OpenMod.Unturned/Environment/Events/UnturnedMoonUpdatedEvent.cs

```diff
@@ -0,0 +1,14 @@
+﻿using OpenMod.Core.Eventing;
+
+namespace OpenMod.Unturned.Environment.Events
+{
+    public class UnturnedMoonUpdatedEvent : Event
```

Comment:
Name needs clarification when this event is emitted (e.g. does it get emitted when it becomes night?) -> maybe rename it to UnturnedFullMoonEvent as we already have an event for listening to night

Reply:
This event is unnecessary and should be under UnturnedDayNightUpdatedEvent as they're called one after the other. This is an instance where I got caught following Unturned's native event scheme.

Reply:
Resolved by 290229f